eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -19,7 +19,7 @@ parameters:
   default: sdk/*/*/*cov_xml.xml
 - name: TimeoutInMinutes
   type: number
-  default: 60
+  default: 120
```

Comment:
Do we really need to move the default timeout? Are the samples adding that much time to the pipeline runs?

Reply:
Yes , actually they are , a couple of the jobs run way over one hour causing the pipeline to fail.

Reply:
OK I guess as a team you guys should decide if you want to be running all the samples if they add that much time or perhaps work on the performance of them.